src/app/cube/usage-stats/usage-stats.component.scss

```diff
@@ -3,7 +3,7 @@
 $row-gap: 25px;
 
 .container {
-  padding: 25px 15px;
+  padding: 30px 50px;
```

Comment:
Why would the removal of bootstrap required a doubling of the padding here? Could the root cause of this be the removal of a 'box-sizing' rule?

Reply:
So it looks like the container has a padding default because it doesn't fill the entire the viewport.